### ISSUE TITLE
Add a top bar on each website

### ISSUE THREAD

**Issue by fheslouin** [ideascube/ansiblecube#7]
The Nginx ngx_http_sub_module will do the job. It is able to replace a HTML code by an other, so we can inject our top bar

**Comment by fheslouin** [comment]
This is still something that can be done...
We just need the HTML top bar

**Comment by fheslouin** [comment]
This feature will be added by Ideascube, issue : https://github.com/ideascube/ideascube/issues/448

**Comment by bochecha** [comment]
https://github.com/ideascube/ideascube/issues/448 is only about the Kiwix top-bar.

This issue seemed to be about all websites?

**Comment by fheslouin** [comment]
That is correct. So far the need did not have been express for other website as we have mostly zim files managed by kiwix. 

I re-open this issue in case we want to have this top bar for other website
